### PR TITLE
Adding 2019 and 2012r2 images

### PR DESCRIPTION
**Pull request by dubb-b**

### CODE REVIEW

File: AWS/windows/windows-2012r2-us-west-2.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "_comment": "Build with `packer build -var-file=windows-2012r2-us-west-2.json windows.json`",
+  "aws_region": "us-west-2",
+  "ami_filter": "Windows_Server-2012-R2_RTM-English-64Bit-Base-*",
+  "ami_owner": "801119661308",
+  "ami_name": "saltstack/base/windows/2012r2",
+  "os_version": "2012r2"
```
Comment (s0undt3ch) [inline AWS/windows/windows-2012r2-us-west-2.json:8]: os_version is not needed, `make` will pass it

File: AWS/windows/windows-2019-us-west-2.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "_comment": "Build with `packer build -var-file=windows-2019-us-west-2.json windows.json`",
+  "aws_region": "us-west-2",
+  "ami_filter": "Windows_Server-2019-English-Full-Base-*",
+  "ami_owner": "801119661308",
+  "ami_name": "saltstack/base/windows/2019",
+  "os_version": "2019"
```
Comment (s0undt3ch) [inline AWS/windows/windows-2019-us-west-2.json:8]: Same here.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

File: AWS/windows/windows-2012r2-us-west-2.json (hunk 1 above)
Comment (dubb-b) [reply AWS/windows/windows-2012r2-us-west-2.json:8]: Ahh I was just using packer command, should we keep it there so all ways work?

Comment (s0undt3ch) [reply AWS/windows/windows-2012r2-us-west-2.json:8]: In that case, lets also add os_name, and to all var files?

Comment (dubb-b) [reply AWS/windows/windows-2012r2-us-west-2.json:8]: @s0undt3ch this should be done now.

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**